External.CustomerReviewsModule/External.CustomerReviewsModule/External.CustomerReviewsModule.Data/Models/CustomerReviewEntity.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using External.CustomerReviewsModule.Core.Models.Search;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace External.CustomerReviewsModule.Data.Model
+{
+    public class CustomerReviewEntity : AuditableEntity
+    {
+        [StringLength(128)]
+        public string AuthorNickname { get; set; }
+
+        [Required]
+        [StringLength(1024)]
+        public string Content { get; set; }
+
+        public bool IsActive { get; set; }
+
+        [Required]
+        [StringLength(128)]
+        public string ProductId { get; set; }
+
+        [Required]
+        public short Value { get; set; }
+
+        public virtual CustomerReview ToModel(CustomerReview customerReview)
+        {
+            if (customerReview == null)
```

Comment:
We definitely need an extension method for such check. May be somewhere in project with common utility classes. @eMazeika what do you think about that?

Reply:
null check? interesting idea, but:
1. it's a sample, let's keep it simple;
2. I can't see any better example in our "real" modules. 
3. Virto coding conventions doesn't mention this either. 
Let's leave it as is.

Reply:
Ok